File: DataStructures/C++/linkedlist.cpp
#include<bits/stdc++.h>
using namespace std;

class Node{
    public:
    int value;
    Node* next;
};

int printingValues(Node* n){
    while (n != NULL){
        cout<<n->value<<" ";
        n = n->next;
    }
    return 0; 
}

int main(){
    Node* first = NULL;
    Node* second = NULL;
    Node* tail = NULL;

    //we declared above: pointers which point to nothing(hence NULL)

    first = new Node();
    second = new Node();
    tail = new Node();

    //we are creating objects in the class LinkedList

    first->value = 3;
    first->next = second;

    //we declared first object with it's value equal to 3 and pointer pointing to the second

    second->value = 5;
    second->next = tail;

    //we declared second object with it's value equal to 5 and pointer pointing the third

    tail->value = 7;
    tail->next = NULL;

    // we declared third object with it's value equal to 7 and pointer still pointing to NULL

    printingValues(first);
}

File: DataStructures/Java/linkedlist.java
class linkedlist{
    Node head;
    static class Node {
        int data;
        Node next;
        Node(int d){
            data = d;
            next = null;
        }
    }

    public void printList(){
        Node n = head;
        while(n!=null){
            System.out.print(n.data + " ");
            n = n.next;
        }
    }
    public static void main(String args[]){
        linkedlist llist = new linkedlist();

        llist.head = new Node(2);
        Node second = new Node(4);
        Node third = new Node(6);

        llist.head.next = second;
        second.next = third;

        llist.printList();
    }
}